### ISSUE TITLE
Names of users are removed!! Simulation Script?

### ISSUE THREAD

**Issue by aminlatifi** [Giveth/feathers-giveth#341]
![image](https://user-images.githubusercontent.com/5684607/103490759-42720780-4e33-11eb-9dad-ee5b85785a2b.png)

**Comment by aminlatifi** [comment]
It happened to my account to and I filled it again.
@mohammadranjbarz I looked in simulation script and didn't see any part related to User entity creating or patching. Am I right?

**Comment by mohammadranjbarz** [comment]
@aminlatifi  Yes you are right I think it's not related to simualtion script, as in develop I ran multiple time these days but the name didn't change

**Comment by mohammadranjbarz** [comment]
@aminlatifi 
I've an idea for knowing if simulation does this or not,
You can edit your name in beta and run simulation script and then again check your name has been changed or not  ( I say checking with your account because I don't have this problem"

**Comment by mohammadranjbarz** [comment]
@aminlatifi 

I investigated on feathers-giveth and simulation, and we don't update the user's name unless in PATCH /users that it's right,
so we can tell that the updated name didn't happen through the feathers

**Comment by GriffGreen** [comment]
Maybe add some logs to the database to understand how this rewrite happened....

**Comment by GriffGreen** [comment]
hopefully the upgrade will fix it

**Comment by aminlatifi** [comment]
Our plan on mongodb service provider (MongoDB Atlas) doesn't include profiling, and logging db commands is part of the profiling service.
We are thinking about hosing DB ourselves for improving performance, if that happen we don't need upgrading our plan on MongoDB Atlas
